eng/pipelines/libraries/base-job.yml

```diff
@@ -95,21 +95,20 @@ jobs:
         - _runtimeArtifactName: ''
         - _runtimeDownloadPath: ''
         - _runtimeArtifactsPathArg: ''
+        - _runtimeConfigurationArg: ''
 
         - ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
           - _runtimeDownloadPath: '$(Build.SourcesDirectory)/artifacts/transport/${{ parameters.runtimeFlavor }}'
-          # Download corelib dependencies for coreclr libraries and not test
-          - ${{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.testScope, '')) }}:
-            - _runtimeArtifactName: 'CoreLib_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.liveRuntimeBuildConfig }}'
+          - _runtimeConfigurationArg: -runtimeConfiguration ${{ parameters.liveRuntimeBuildConfig }}
```

Comment:
Can you use the short alias? It was introduced yesterday, `-rc` -- that will make the call in the logs easier to parse/read.

Reply:
done